### PR TITLE
Item hover info panel in inventory

### PR DESCRIPTION
**Pull request by ajrb**
Enabled by a setting, disabled by default.
Thanks to Nystul for the text scaling.

### CODE REVIEW

**Comment by ajrb** [comment]
Should mention I disabled potion popup info, temporarily. Will be reimplementing using the macro handler system.

File: Assets/Scripts/SettingsManager.cs [hunk 1]
```diff
@@ -155,6 +156,7 @@ public void LoadSettings()
             ToolTipDelayInSeconds = GetFloat(sectionGUI, "ToolTipDelayInSeconds", 0, 10);
             ToolTipBackgroundColor = GetColor(sectionGUI, "ToolTipBackgroundColor", DaggerfallUI.DaggerfallUnityDefaultToolTipBackgroundColor);
             ToolTipTextColor = GetColor(sectionGUI, "ToolTipTextColor", DaggerfallUI.DaggerfallUnityDefaultToolTipTextColor);
+            EnableInventoryInfoPanel = GetBool(sectionGUI, "EnableInventoryInfoPanel");
```
Comment (Interkarma) [inline Assets/Scripts/SettingsManager.cs]: Don't forget the matching SaveSettings() for this value. :)

**Comment by Interkarma** [comment]
This looks good, happy to have this change. Up to you about the potion popup, I think it's OK to just let this be non-functional for a while.

**Comment by Interkarma** [comment]
This works really well - good job! :)

The only refinement I'd suggest is for the info panel to clear when mouse not hovering on an item. It feels a touch strange to have the item info still displaying when mouse no longer over an item.

But otherwise feels very modern and snappy.

File: Assets/Scripts/SettingsManager.cs (hunk 1 above)
Comment (ajrb) [reply Assets/Scripts/SettingsManager.cs]: Is that so it can be added to the settings window?

**Comment by Dimillian** [comment]
I love it, I was thinking about that ! 
For now the settings is not exposed into the launch window, also if enable I think it should disable the tooltips. And yes, it should clear up on mouse out.

**Pull request merged**